packages/web-components/src/components/va-date/test/va-date.e2e.ts

```diff
@@ -87,7 +89,7 @@ describe('va-date', () => {
       await handleYear.press('Tab');
 
       await page.waitForChanges();
-      expect(date.getAttribute('error')).toEqual("Please enter a year between 1900 and 2122");
+      expect(date.getAttribute('error')).toEqual(`Please enter a year between 1900 and ${maxYear}`);
```

Comment:
The `va-date` and `va-memorable-date` components are not related to this PR but we needed to update the tests that had the value "2122" hard coded as part of the validation for it to pass in the new year.